src/Sylius/Bundle/CoreBundle/PaymentRequest/Processor/Offline/AfterCaptureProcessor.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\PaymentRequest\Processor\Offline;
+
+use SM\Factory\FactoryInterface as StateMachineFactoryInterface;
+use Sylius\Abstraction\StateMachine\StateMachineInterface;
+use Sylius\Component\Payment\Model\PaymentRequestInterface;
+use Sylius\Component\Payment\PaymentTransitions;
+
+final class AfterCaptureProcessor implements AfterCaptureProcessorInterface
```

Comment:
What is the main idea behind splitting capture into normal and after? If I see correctly, we always trigger both of them

Reply:
You are right, I wanted to split those two but it's not required.